lib/engine/game/g_1848/game.rb

```diff
@@ -794,7 +833,8 @@ def edge_is_a_border(edge)
         end
 
         def revenue_for(route, stops)
-          k_sum = stops.count { |rl| rl.hex.tile.label.to_s == 'K' }
+          k_sum = stops.count { |rl| rl.hex&.tile&.label&.to_s == 'K' || rl.hex&.tile&.future_label&.label.to_s == 'K' }
```

Comment:
Thought: I wonder if future labels would be better if they were a form of label, that just applied later on with separate rendering rules. I'm not sure how often logic like this will be needed to distinguish between future labels and labels

Reply:
It is a bit of a future label situation, the K is on the map, but not on the yellow tile, but is on the green tile.
Normal label won't work (or will require extra logic, which i removed so i can use future label instead).
Visually, it's clear that those are k tiles, i'm not sure if this needs rework

Reply:
agreed, but i don't want to touch the future label code at this point

Reply:
yeah, i don't know of a game that would use future labels, but not treat them as regular label, but maybe there are some out there.